thread.c

```diff
@@ -816,87 +814,87 @@ thread_start_func_2(rb_thread_t *th, VALUE *stack_start)
     th->ec->machine.stack_start = STACK_DIR_UPPER(vm_stack + size, vm_stack);
     th->ec->machine.stack_maxsize -= size * sizeof(VALUE);
 
-    {
-	thread_debug("thread start (get lock): %p\n", (void *)th);
+    thread_debug("thread start (get lock): %p\n", (void *)th);
```

Comment:
The following diffs by removing `{ ... }`?

Reply:
`w=1` helps me.

Reply:
Mixing unnecessary cosmetic changes is not a good idea, I think.

Reply:
The indentation makes it incredibly hard to edit this code, so I gave up and reformatted all of it.